Meta/Web Foundations team responsibilities.md

```diff
@@ -0,0 +1,67 @@
+
+Additionally, the Web Foundation team should be at least a "keep informed" stakeholder on any project that involves a key extension or change to our typical process of developing for the Web, and for any project that involves significant and novel use of Web technologies not done elsewhere in Shopify.
+
+## Stewards of common tools, libraries, and best practices
+
+At Shopify, we do not believe that every project should have completely free-reign in deciding their product’s tech stack. We expect projects to use our common developer tools, deploy systems, and production environments. We also make "picks" at higher levels in the stack that we believe are better served by consistency than developer choice, such as our preference for Rails as an application framework, or our preference for MySQL as the primary data store. These choices are often made in an effort to provide a minimum level of quality from all the products we create.
+
+The Web Foundation team applies this same principle of having sensible "picks" to developer tools, application libraries, and best practices specific to the Web platform. However, we are mindful of not creating "thin clients" — that is, leaving no decisions to application teams about what their application should look like.
+
+It is useful to consider the separation of powers between federal and provincial governments, as this is a similar model to how the Web Foundation team considers topics in which to involve themselves. In Canada, the federal government delegates the responsibility for overseeing schools, hospitals, and property rights to provincial governments, while retaining powers that they are best positioned to address, such as national defence and currency. Similarly, the Web Foundations team is responsible for guiding decisions on topics that are important for all applications, but delegates most of the finer points to application teams.
+
+### Criteria for Web Foundation involvement
+
+The above description can be seen as overly broad, so the Web Foundation team uses the following criteria in deciding whether it should take a role in deciding best practices or preferred tools for a particular topic.
+
+The following criteria must **always** be met for the Web Foundation team to get involved:
+
+* It **must not** interfere with a team’s ability to deliver value to their user. This has two implications:
+  1. The Web Foundation team only builds and maintains libraries that are either non-visual for the end user, or which operate in line with our design system, Polaris.
+  2. Any team feeling that our recommended best practices or libraries prevent them from being capable of delivering a user-facing feature should ignore the recommendation and bring this up with us.
+* It **must** be a topic that affects a majority of applications. The Web Foundation team will not build libraries for that benefit only a small handful of projects.
```

Comment:
`for that` -> `that`

Reply:
Also, what do we mean by `small handful`? Don't we build such tools for `core` on occasion? Perhaps rephrase this to mention scope rather than number of projects?